src/smart_qq_bot/http_client.py

```diff
@@ -24,8 +24,8 @@ class HttpClient(object):
     # urllib2.install_opener(_req)
 
     def __init__(self, load_cookie=False, cookie_file=COOKIE_FILE):
-        if not os.path.isdir("./cookie"):
-            os.mkdir("./cookie")
+        if not os.path.isdir(os.path.dirname(cookie_file)):
```

Comment:
I had tested the feather, **all function works great!** 
the http server port is assign by docker, that may need user to use `docker ps` to find that port to use, maybe we can do some improvement on user guide later ;D

May I merge this pull request :D